app/tools/android/test_data/extensions/echoJsStubGenExtension/src/com/example/jsStub/Echo.java

```diff
@@ -0,0 +1,78 @@
+package com.example.jsStub;
+
+import android.util.Log;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.xwalk.app.runtime.extension.*;
+
+/*
+ * Example for JavaScript stub auto-generation feature on Android.
+ */
+public class Echo extends XWalkExtensionClient {
+
+  // Expose a read only JS property.
+  @JsAPI
+  public String prefix = "From java:";
```

Comment:
point the "isWritable" property of the annotation, for example: @JsAPI(isWritable = true), the default value is false.